src/app/releasemanager.cpp

```diff
@@ -418,10 +419,12 @@ void Release::setLocalFile(const QString &path) {
     if (m_source != LOCAL)
         return;
     QFileInfo info(QUrl(path).toLocalFile());
-
+        
     if (!info.exists()) {
-        mCritical() << path << "doesn't exist";
-        return;
+        if (info.path() != "") {
```

Comment:
This looks odd. You also don't want to continue later adding a new ReleaseVersion to the model.

Write a separate function, something like "resetLocalFile()" which will just remove a ReleaseVersion and emits versionsChanged() and selectedVersionChanged();.